app/services/agent_interaction_service.py

```
@@ -965,34 +968,38 @@ async def _resolve_conflict_escalate(self, conflict_id: UUID):
-                        "event_type": "agent_error",
-                        "message": f"Conflict escalated: {conflict.get('conflict_type', 'unknown')}",
-                        "event_data": {
-                            "conflict_id": str(conflict_id),
-                            "conflict_type": conflict.get("conflict_type"),
-                            "requires_manual_intervention": True
-                        },
-                        "priority": 10,  # Highest priority
-                        "metadata": {"escalated_conflict_id": str(conflict_id)}
-                    }).execute()
+            if not crew_response.data or not crew_response.data.get("agent_ids"):
+                raise ValueError(f"No agents found for crew {crew_id} during escalation")
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)